freqtrade/plot/plotting.py

```diff
@@ -431,44 +460,15 @@ def generate_candlestick_graph(pair: str, data: pd.DataFrame, trades: pd.DataFra
     )
     fig.add_trace(candles, 1, 1)
 
-    # TODO-lev: Needs short equivalent
-    if 'enter_long' in data.columns:
-        df_buy = data[data['enter_long'] == 1]
-        if len(df_buy) > 0:
-            buys = go.Scatter(
-                x=df_buy.date,
-                y=df_buy.close,
-                mode='markers',
-                name='buy',
-                marker=dict(
-                    symbol='triangle-up-dot',
-                    size=9,
-                    line=dict(width=1),
-                    color='green',
-                )
-            )
-            fig.add_trace(buys, 1, 1)
-        else:
-            logger.warning("No buy-signals found.")
-
-    if 'exit_long' in data.columns:
-        df_sell = data[data['exit_long'] == 1]
-        if len(df_sell) > 0:
-            sells = go.Scatter(
-                x=df_sell.date,
-                y=df_sell.close,
-                mode='markers',
-                name='sell',
-                marker=dict(
-                    symbol='triangle-down-dot',
-                    size=9,
-                    line=dict(width=1),
-                    color='red',
-                )
-            )
-            fig.add_trace(sells, 1, 1)
-        else:
-            logger.warning("No sell-signals found.")
+    longs = create_scatter(data, 'enter_long', 'green', 'up')
+    exit_longs = create_scatter(data, 'exit_long', 'red', 'down')
```

Comment:
To be honest, i'd split the changes to the plotting modules out of here to a separate PR (if you're keen on completing them - for me, i could live with not having futures support for plotting for the moment).

these changes suggest it's working - but it really isn't, and produced results are wrong.

Reply:
I reverted the changes, the commit( sha: 0abba7f9b7299ba3c45df6a2ba6e35ad6a19c5a0, message: "plot.generate_candlestick_graph Added short equivelent, separating plotting scatter creation to a function ) is there if you want to use it, I've never even used plotting so I won't try to figure it out this way

Reply:
Thanks - i've extracted the "forward" commit to the branch `short_plot` (otherwise i'm 100% sure we'll lose it).
I only know it ain't working - i didn't look into the details yet as to why (i suspect it's loading the wrong data - but didn't check further.